skimage/filters/edges.py

```diff
@@ -12,8 +12,8 @@
 import numpy as np
 from .. import img_as_float
 from .._shared.utils import check_nD
-from scipy.ndimage import convolve, binary_erosion, generate_binary_structure
-
+from scipy.ndimage import binary_erosion, generate_binary_structure
+from scipy.ndimage import convolve
```

Comment:
@Dalordish can you undo changes to this file?

Reply:
Done